src/Raven.Server/Documents/DocumentsStorage.cs

```diff
@@ -238,6 +238,7 @@ public void Initialize(bool generateNewDatabaseId = false)
             options.PrefetchResetThreshold = DocumentDatabase.Configuration.Storage.PrefetchResetThreshold.GetValue(SizeUnit.Bytes);
             options.SyncJournalsCountThreshold = DocumentDatabase.Configuration.Storage.SyncJournalsCountThreshold;
             options.IgnoreInvalidJournalErrors = DocumentDatabase.Configuration.Storage.IgnoreInvalidJournalErrors;
+            options.SkipChecksumValidationOnDatabaseLoading = DocumentDatabase.Configuration.Storage.SkipChecksumValidationOnDatabaseLoading;
```

Comment:
We need to add this to all the storages that we have (configuration, indexes, etc)